packages/react-scripts/config/webpack.config.js

```diff
@@ -86,7 +86,7 @@ module.exports = function (webpackEnv) {
   // Get environment variables to inject into our app.
   const env = getClientEnvironment(paths.publicUrlOrPath.slice(0, -1));
 
-  const shouldUseReactRefresh = env.raw.FAST_REFRESH;
+  const shouldUseReactRefresh = env.raw.FAST_REFRESH !== 'false';
```

Comment:
This would be a boolean wouldn't it?

Reply:
It's a string when it comes from environment variables - I've been caught on this before 😆 

```sh
SOME_VAR=true myapp
```

```js
console.log(process.env.SOME_VAR) // 'true'
```

Reply:
We're using our env object here from https://github.com/iansu/create-react-app/blob/b7e99866f734547818aabde017bbb98d4232c79a/packages/react-scripts/config/env.js#L100